bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoBindingConstants.java

```diff
@@ -57,7 +57,8 @@
     public static final String CHANNEL_ONTOTAL = "onTotal";
     public static final String CHANNEL_TIMESPAN = "timespan";
     public static final String CHANNEL_AVERAGEPOWER = "averagePower";
-    public static final String CHANNEL_CURRENTPOWER = "currentPower";
+    public static final String CHANNEL_CURRENT_POWER = "currentPower";
+    public static final String CHANNEL_CURRENT_POWER_RAW = "currentPowerRaw";
```

Comment:
IMHO, this is better with _ for readability but as all the other constants are without `_`, maybe it would be better to keep something homogeneous ?

Reply:
All constants renamed for readability and consistency.